src/platform/qt/scripting/AutorunScriptModel.h

```diff
@@ -17,42 +17,42 @@ Q_OBJECT
 
 public:
 	struct ScriptInfo {
+		static const uint8_t VERSION = 1;
```

Comment:
Though I doubt it will ever exceed 255, I don't think it's really worth artificially limiting it to that. Perhaps uint16_t.

Reply:
Sure, that seems fair.